### PR TITLE
Maths Magicians

### PR DESCRIPTION
**Pull request by Donmark2k**
### Highlights 🥇

**Maths Magicians**

**Learning objectives**

- Set up a React app.
- Use React documentation.
- Use React components.
- Use React props.
- Use React state.

**What I Have Done:**

- Added linters. The [react & redux linter](https://github.com/microverseinc/linters-config/tree/master/react-redux).
- I create a git flow.
- I created my-app folder
- I created a calculator UI
- I deleted all the boilerplate from Create React App (text, images, styles).

**Followed Rules:**

- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
- Followed [Git Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)
- Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- Followed the list of [best practices for react & redux](https://github.com/microverseinc/linters-config/tree/master/react-redux).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+/* eslint-disable */
+
 import './App.css';
-import Calculator from './components/calculator';
+import Calculator from './components/Calculator';
```
Comment (FrederickMih) [inline src/App.js]: - [ ] The imported **Calculato**r does not match the corresponding file name you're importing. It should be lower case **_c_**calculator and not **_C_**alculator

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi 🙋 @Donmark2k   , 


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights </img>

- Great commit history ✔️ 
- PR is professional ✔️ 
- Linters checks passed ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Donmark2k) [reply src/App.js:4]: The file name is Calculator.js

**Review by DammyShittu — APPROVED** [review]
Hi @Donmark2k,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**